pkg/apis/batch/types.go

```diff
@@ -138,6 +138,18 @@ const (
 	PodFailurePolicyOnExitCodesOpNotIn PodFailurePolicyOnExitCodesOperator = "NotIn"
 )
 
+// PodReplacementPolicy specifies the policy for creating pod replacements.
+// +enum
+type PodReplacementPolicy string
+
+const (
+	// TerminatingOrFailed means that we recreate pods
+	// when they are terminating (has a metadata.deletionTimestamp) or failed.
+	TerminatingOrFailed PodReplacementPolicy = "TerminatingOrFailed"
+	// Failed means to recreate pods when they are marked as failed.
```

Comment:
these definitions should be the same as in PodReplacementPolicy field, no?

Reply:
Good call!  Pushed up a fix.